bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/luckperms/LuckPermsTrack.java

```diff
@@ -0,0 +1,148 @@
+package com.denizenscript.depenizen.bukkit.objects.luckperms;
+
+import me.lucko.luckperms.LuckPerms;
+import me.lucko.luckperms.api.Group;
+import me.lucko.luckperms.api.LuckPermsApi;
+import me.lucko.luckperms.api.Track;
+import net.aufdemrand.denizencore.objects.Element;
+import net.aufdemrand.denizencore.objects.Fetchable;
+import net.aufdemrand.denizencore.objects.dList;
+import net.aufdemrand.denizencore.objects.dObject;
+import net.aufdemrand.denizencore.tags.Attribute;
+import net.aufdemrand.denizencore.tags.TagContext;
+
+public class LuckPermsTrack implements dObject {
+    /////////////////////
+    //   OBJECT FETCHER
+    /////////////////
+
+    public static LuckPermsTrack valueOf(String string) {
+        return valueOf(string, null);
+    }
+
+    @Fetchable("luckpermstrack")
+    public static LuckPermsTrack valueOf(String string, TagContext context) {
+        if (string == null) {
+            return null;
+        }
+
+        ////////
+        // Match plotsquaredplot name
+
+        string = string.replace("luckpermstrack@", "");
+        try {
+            LuckPermsApi api = LuckPerms.getApi();
+            Track track = api.getTrack(string);
+            if (track == null) {
+                return null;
+            }
+            return new LuckPermsTrack(track);
+        }
+        catch (Throwable e) {
+            // Do nothing.
```

Comment:
Why!

Reply:
![image](https://user-images.githubusercontent.com/8504075/37690373-c86e4edc-2caa-11e8-8827-3efe5bd7346d.png)
Is this fine?

Reply:
No. HANDLE THE EXCEPTION. Don't just ignore it.